src/include/sof/audio/codec_adapter/interfaces.h

```diff
@@ -43,15 +43,15 @@ static struct codec_interface interfaces[] = {
 	},
 #endif /* CONFIG_CADENCE_CODEC */
 
-#ifdef CONFIG_DUMMY_CODEC
+#ifdef CONFIG_PASSTHROUGH_CODEC
 	{
-		.id = DUMMY_ID, /** dummy interface */
-		.init  = dummy_codec_init,
-		.prepare = dummy_codec_prepare,
-		.process = dummy_codec_process,
-		.apply_config = dummy_codec_apply_config,
-		.reset = dummy_codec_reset,
-		.free = dummy_codec_free
+		.id = PASSTHROUGH_ID, /** passthrough interface */
+		.init  = passthrough_codec_init,
+		.prepare = passthrough_codec_prepare,
+		.process = passthrough_codec_process,
+		.apply_config = passthrough_codec_apply_config,
+		.reset = passthrough_codec_reset,
+		.free = passthrough_codec_free
 	},
 #endif /* CONFIG_DUMMY_CODEC */
```

Comment:
in the comment please also rename to `CONFIG_PASSTHROUGH_CODEC`

Reply:
@dbaluta please feel free to amend and add your signoff, otherwise we will need to close this PR so it can be pulled from my branch or will need to give me access to your repo which seem like too much effort.

Reply:
@cujomalainey thanks! Done!